src/conf_mode/protocols_static.py

```diff
@@ -98,6 +111,15 @@ def generate(static):
     return None
 
 def apply(static):
+    ## Put routing table names in /etc/iproute2/rt_tables
+    with open("/etc/iproute2/rt_tables", 'w') as f:
```

Comment:
Maybe it is better don't touch `rt_tables` and use `rt_tables.d`?
```
vyos@r1# sudo cat /etc/iproute2/rt_tables.d/mytable.conf
2499    mytable
[edit]
vyos@r1# 
[edit]
vyos@r1# sudo ip route show table mytable
default via 10.217.80.254 dev eth4 
[edit]
vyos@r1# 
```

Reply:
Oh, that's a fair point. If it works as expected, that would be a cleaner approach. I'll test it today and set it to write to something like `/etc/iproute2/rt_tables.d/vyos.conf`.

@jestabro Do you agree?

Reply:
we do something like this for vrf
https://github.com/vyos/vyos-1x/blob/28c7960b6cc9bb05fafb0f2b8659b169b13c0071/src/conf_mode/vrf.py#L38

Reply:
Agreed; great suggestion, if it works as advertised, and the vrf example would seem to indicate that it does.